### PR TITLE
RPM spec: add typing extension

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: package/python-kiwi-spec-template [hunk 1]
```diff
@@ -374,6 +374,7 @@ Requires:       python%{python3_pkgversion}-docopt
 Requires:       python%{python3_pkgversion}-lxml
 Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-setuptools
+Requires:       (python%{python3_pkgversion}-typing_extensions if python%{python3_pkgversion} < 3.8)
```
Comment (Conan-Kudo) [inline package/python-kiwi-spec-template:377]: The package name is `python3-typing-extensions` in RH/Fedora and Debian/Ubuntu.

And conditional dependencies don't work on Debian/Ubuntu at all, so this needs to be handled correctly for that too.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

**Review by Conan-Kudo — APPROVED** [review]

**Comment by dcermak** [comment]
> @dcermak still in draft mode ?

To prevent a premature merge :sweat_smile:

Is it good on OBS?

**Comment by schaefi** [comment]
> To prevent a premature merge 

:) we usually set the "do-not-merge" flag to indicate this

> Is it good on OBS?

still running, stay tuned I will respond back

Thanks

**Comment by schaefi** [comment]
All green again, thanks a ton for fixing this

**Pull request merged**